mlflow/environment_variables.py

```diff
@@ -604,3 +604,10 @@ def get(self):
 MLFLOW_REQUIREMENTS_INFERENCE_RAISE_ERRORS = _BooleanEnvironmentVariable(
     "MLFLOW_REQUIREMENTS_INFERENCE_RAISE_ERRORS", False
 )
+
+#: Private configuration option.
+#: Whether to use mlflow langchain tracer for rag tracing. This should be set in model
+#: serving for rag models.
+_USE_MLFLOW_LANGCHAIN_TRACER_FOR_RAG_TRACING = _BooleanEnvironmentVariable(
+    "USE_MLFLOW_LANGCHAIN_TRACER_FOR_RAG_TRACING", False
```

Comment:
```suggestion
    "MLFLOW_USE_LANGCHAIN_TRACER_FOR_RAG_TRACING", False
```

can we swap `USE` and `MLFLOW`?

Reply:
I'm ok with the current name if `MLFLOW_USE_LANGCHAIN_TRACER_FOR_RAG_TRACING` sounds confusing.

Reply:
I'll go with the current name as Corey suggested, we'll probably drop this variable if downstream consumers supports new outputs & inputs in the future

Reply:
sounds good!